classification_analysis.ipynb

```diff
@@ -229,9 +311,10 @@
     "                    continue\n",
```

Comment:
here's where we skip the tree entries that are not in the correct role. You should be able to add logging just before this and verify that there are in fact 6 runs but only 3 have the correct role.